<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aggregation</class>
 <widget class="QWidget" name="Aggregation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layout_main">
   <item>
    <widget class="AttributeSelector" name="widget_group_attribute" native="true"/>
   </item>
   <item>
    <widget class="AttributeSelector" name="widget_ident_property" native="true"/>
   </item>
   <item>
    <widget class="FileSelector" name="widget_import" native="true"/>
   </item>
   <item>
    <widget class="FileSelector" name="widget_export" native="true"/>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>620</width>
        <height>70</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout_progress_bar"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelector</class>
   <extends>QWidget</extends>
   <header>som_gui.module.util.ui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AttributeSelector</class>
   <extends>QWidget</extends>
   <header>som_gui.module.util.ui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
